src/trio/testing/_raises_group.py

```diff
@@ -503,6 +605,25 @@ def __enter__(self) -> ExceptionInfo[BaseExceptionGroup[BaseException]]:
         )
         return self.excinfo
 
+    def __str__(self) -> str:
+        return self.__repr__()
+
+    def __repr__(self) -> str:
+        reqs = [
+            e.__name__ if isinstance(e, type) else repr(e)
```

Comment:
I would expect a qualified name is nicer. But honestly, not completely sure.

Reply:
I think we're better of with an unqualified name here. Maybe not super-"repr"-y, but we don't want/need `my_package.exceptions.MyError` in our long messy tables, and in the rare case where that disambiguation is needed they can get it from looking at the proper backtrace.